stats/stats-server/tests/it/mock_blockscout_simple/stats_not_indexed.rs

```diff
@@ -3,141 +3,116 @@
 //! - stats server is fully enabled & updated as much as possible
 //!     with not indexed blockscout
 
-use std::time::Duration;
-
-use blockscout_service_launcher::{
-    test_database::TestDbGuard,
-    test_server::{init_server, send_get_request},
-};
+use blockscout_service_launcher::{launcher::GracefulShutdownHandler, test_server::init_server};
 use futures::FutureExt;
-use itertools::Itertools;
 use pretty_assertions::assert_eq;
-use stats::tests::{init_db::init_db, mock_blockscout::mock_blockscout_api};
-use stats_proto::blockscout::stats::{
-    v1 as proto_v1,
-    v1::{health_check_response::ServingStatus, Counters, HealthCheckResponse},
+use stats::tests::{
+    init_db::init_db,
+    mock_blockscout::{mock_blockscout_api, user_ops_status_response_json},
 };
+use stats_proto::blockscout::stats::v1 as proto_v1;
 use stats_server::{
     auth::{ApiKey, API_KEY_NAME},
     stats,
 };
-use tokio::{task::JoinSet, time::sleep};
+use tokio::task::JoinSet;
 use url::Url;
 use wiremock::ResponseTemplate;
 
-use super::common_tests::{test_contracts_page_ok, test_main_page_ok, test_transactions_page_ok};
-use crate::common::{
-    enabled_resolutions, get_test_stats_settings, run_consolidated_tests, send_arbitrary_request,
-    setup_single_key, sorted_vec,
+use super::common_tests::{
+    test_contracts_page_ok, test_counters_ok, test_lines_ok, test_main_page_ok,
+    test_transactions_page_ok,
+};
+use crate::{
+    common::{
+        get_test_stats_settings, run_consolidated_tests, send_arbitrary_request, setup_single_key,
+        wait_for_subset_to_update, ChartSubset,
+    },
+    it::mock_blockscout_simple::get_mock_blockscout,
 };
 
-pub async fn run_tests_with_charts_uninitialized(blockscout_db: TestDbGuard) {
-    let test_name = "run_tests_with_charts_uninitialized";
+#[tokio::test]
+#[ignore = "needs database"]
+pub async fn run_tests_with_nothing_indexed() {
+    let test_name = "run_tests_with_nothing_indexed";
+    let _ = tracing_subscriber::fmt::try_init();
     let stats_db = init_db(test_name).await;
-    let blockscout_api = mock_blockscout_api(ResponseTemplate::new(200).set_body_string(
-        r#"{
-            "finished_indexing": false,
-            "finished_indexing_blocks": false,
-            "indexed_blocks_ratio": "0.00",
-            "indexed_internal_transactions_ratio": null
-        }"#,
-    ))
+    let blockscout_db = get_mock_blockscout().await;
+    let blockscout_api = mock_blockscout_api(
+        ResponseTemplate::new(200).set_body_string(
```

Comment:
using [set_body_json](https://docs.rs/wiremock/latest/wiremock/struct.ResponseTemplate.html#method.set_body_json) you can use serde_json::json!() to enforce json structure :)

Reply:
wow, it's nice; thx!